### ISSUE TITLE
Export in Zotero broken

### ISSUE THREAD

**Issue by schaeferRCOHR** [Juris-M/zotero-odf-scan-plugin#33]
As of today, after a Zotero update (5.0.79) when I try to scan a file (.odt), it only allows to export as .rtf and the resulting file is corrupted. If I convert to rtf before and scan the file, it basically returns the file unchanged (i.e. text codes are not converted to Zotero field codes).

Given that the Better Bibtext Expansion also stopped working after the update, I assume it is related to some code update in Zotero.
https://forums.zotero.org/discussion/80285/better-bibtex-5-1-165-5-1-166-an-error-has-occurred-please-restart-zotero-fixed-in-5-1-167

**Comment by schaeferRCOHR** [comment]
Seems to occur as well for other users https://forums.zotero.org/discussion/comment/344150#Comment_344150

**Comment by dstillman** [comment]
This will be fixed by https://github.com/Juris-M/zotero-odf-scan-plugin/pull/34.

(The BBT update was unrelated to the Zotero update. BBT had already been updated for the 5.0.78 changes months earlier. There was just a broken BBT update around the same time.)

**Comment by schaeferRCOHR** [comment]
Great - thanks for the work!

**Comment by fbennett** [comment]
@retorquere Would it be possible for you to push the build? I've merged Dan's patch, but my system here is on Python 3.5, the notes say Python 3.7 is required, and I'm hesitant to tangle with upgrading it just now.

**Comment by dstillman** [comment]
No, please hold — I found some more breakage.

**Comment by retorquere** [comment]
@fbennett I can easily make it work with Py3.5. I have a crazy day ahead so the next 10 hours or so I'm unavailable, but after that I'll make it happen.

**Comment by fbennett** [comment]
Thanks!

**Comment by roycemorris** [comment]
Hi all. I have no understanding of technology. But I know when I press that button in Zotero my bibliography works. If someone can fix this I would be eternally grateful!

**Comment by fbennett** [comment]
Coming soon. Have some errands and cooking to take care of first, but it
will be out later today.

Frank Bennett

**Comment by fbennett** [comment]
@roycemorris It should be up and working now, at https://github.com/Juris-M/zotero-odf-scan-plugin/releases/latest

**Comment by michaelaye** [comment]
I do not get a "Scannable Cite" as an export item using 2.0.43? LibreOffice is installed.

I'm using the Zotero beta b/c of the Safari plugin, could that be an issue?

**Comment by fbennett** [comment]
Yes, sorry about that. The Abbrevs Filter is broken in the 5.0.80m3
release. Just fixed it, and the fixit release will be up in a few yours.

**Comment by fbennett** [comment]
Wait, that response was misdirected. Will check this in the latest ODF Scan
release, and get back.

**Comment by nchachereau** [comment]
> I do not get a "Scannable Cite" as an export item using 2.0.43? LibreOffice is installed.

I installed the extension today, and had the same problem. Simply copying the file `Scannable Cite.js` into my translation folder fixed the problem.

@fbennett : I guess there is a problem with the installation of the package? The file does not get copied for some reason?

@michaelaye and any one else with this problem, until it gets fixed in the extension, here how to do what I did:

- download the file [Scannable Cite.js](https://raw.githubusercontent.com/Juris-M/zotero-odf-scan-plugin/master/resource/translators/Scannable%20Cite.js)
- open the folder that contains your Zotero data (here's what the [official documentation](https://www.zotero.org/support/zotero_data#locating_your_zotero_data) says: “The easiest and most reliable way to find your Zotero data is by clicking the “*Show Data Directory*” button in the *Advanced* tab of your Zotero Preferences window.”)
- copy the downloaded `Scannable Cite.js` file into the `translators` folder inside the folder you just opened.

I hope this works for others as well (I think it should.)
